test/Sema/import_package_module_from_sdk.swift

```diff
@@ -0,0 +1,43 @@
+// RUN: %empty-directory(%t)
+// RUN: split-file %s %t
+
+// --- Prepare SDK (.swiftmodule).
+// RUN: %empty-directory(%t/SDK)
+// RUN: mkdir -p %t/SDK/Frameworks/LibInSDK.framework/Modules/LibInSDK.swiftmodule
+// RUN: %target-swift-frontend \
+// RUN:     -emit-module \
+// RUN:     -module-name LibInSDK \
+// RUN:     -package-name libPkg \
+// RUN:     -o %t/SDK/Frameworks/LibInSDK.framework/Modules/LibInSDK.swiftmodule/%module-target-triple.swiftmodule \
+// RUN:     -swift-version 5 \
+// RUN:     %t/Lib.swift
+
+// RUN: test -f %t/SDK/Frameworks/LibInSDK.framework/Modules/LibInSDK.swiftmodule/%module-target-triple.swiftmodule
+
+// RUN: %target-swift-frontend -typecheck %t/Client1.swift -package-name libPkg -sdk %t/SDK -I %t -I %t/SDK/Frameworks/LibInSDK.framework/Modules 2> %t/result.output
+// RUN: %FileCheck %s < %t/result.output
+// CHECK: warning: module 'LibInSDK' is in package 'libPkg' but was loaded from SDK; modules of the same package are expected to be loaded from local build directory: {{.*}}/SDK/Frameworks/LibInSDK.framework/Modules/LibInSDK.swiftmodule/{{.*}}.swiftmodule
```

Comment:
> modules of the same package are expected to be loaded from local build directory

Adds confusion IMO. It's not strictly true, since I could have various build directories and that would still be fine. It also begs the question as to what *is* the "local build directory" 😅.

I think just the prior warning is good enough (ie. `module 'mod' is in package 'package' but was loaded from the SDK`). Or alternatively `SDK module 'mod' should not be included in user-defined package 'package'`, which IIUC is more what the warning is about?

Reply:
The latter sounds good; will update